src/main/resources/public/src/views/Dashboard.vue

```diff
@@ -1,13 +1,54 @@
 <template>
-    
+    <div id="container" class="container">
+        <h2>Dashboard</h2>
+        <table class="table table-sm">
+            <thead class="thead-dark">
+                <tr>
+                    <th>Nom</th>
+                    <th>DC Model</th>
+                    <th>Organization</th>
+                    <th>State</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr v-for="dataRequest in dataRequests">
```

Comment:
Event if it's not actually a component yet I supposed it will become one in the future. Maybe we should already add the v-bind:key now to avoid forgetting it later. (Maybe something like : 
```
<tr v-for="dataRequest in dataRequests" v-bind:dataRequest="dataRequest" v-bind:key="dataRequest.id">
     <td>{{ dataRequest.nom }}</td>
     <td>{{ dataRequest.model }}</td>
     <td>{{ dataRequest.organization }}</td>
     <td>{{ dataRequest.state }}</td>
</tr>
```

Reply:
I don't think we'll forget to add them, because it will not work if we forget them

Reply:
True